docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -0,0 +1,518 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-7-installing-web-console-on-openshift_{context}"]
+
+= Installing the {ProductName} {WebName}
+
+You install the {ProductName} ({ProductShortName}) {WebName} as part of the process of installing the {ProductShortName} Operator on the OpenShift Container Platform.
+
+The {ProductShortName} Operator is a structural layer that manages resources deployed on Kubernetes (database, front end, back end) to automatically create an {ProductShortName} instance.
```

Comment:
@sbeskin-redhat - what do you think `front end` or `front-end`?
Thanks

Reply:
@anarnold97 
front end

Reply:
cool